### PR TITLE
[1LP][RFR] Reports collections conversion

### PR DESCRIPTION
**Pull request by quarckster**
{{pytest: -v -k "test_report_view or test_reports_generate_report or test_infra_report_page_per_item or test_providers_summary or test_cluster_relationships or test_operations_vm_on or test_datastores_summary or test_custom_report_crud or test_reports_generate_report or test_shuffle_top_level or test_shuffle_first_level or test_charge_report_filter_owner or test_custom_vm_report"}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 60d94b76f7a6e3f0d7277b8e417b71c45d5fb30a:

`cfme/tests/intelligence/reports/test_generate_report.py`:
- :heavy_exclamation_mark: Line 52:1: **W293** *blank line contains whitespace*
- :heavy_exclamation_mark: Line 56:28: **W292** *no newline at end of file*

`cfme/tests/intelligence/reports/test_views.py`:
- :red_circle: Line 29:29: **F821** *undefined name 'true'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by quarckster** [comment]
PRT failures are not related to the changes.

**Comment by izapolsk** [comment]
@quarckster, please rebase

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit eb1113e2acaea775e9ccac839eff54ad88c24be8:

`setup.py`:
- :bangbang: Line 173:75: **E999** *SyntaxError: invalid syntax*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/intelligence/reports/reports.py [hunk 1]
```diff
@@ -175,65 +223,58 @@ def is_displayed(self):
...
+    show_event_count = attr.ib(default=None)
+    summary = attr.ib(default=None)
+    charts = attr.ib(default=None)
+    timeline = attr.ib(default=None)
+    is_candu = attr.ib(default=False)
+
+    def __attrs_post_init__(self):
+        self._collections = {'saved_reports': SavedReportsCollection}
+
+    @company_name.default
```
Comment (mshriver) [inline cfme/intelligence/reports/reports.py]: TIL - cool.

File: cfme/intelligence/reports/reports.py [hunk 2]
```diff
@@ -414,12 +432,58 @@ def data(self):
...
+            "Delete this Saved Report from the Database",
+            handle_alert=not cancel
+        )
+        if cancel:
+            assert view.is_displayed
+            view.flash.assert_no_error()
+        else:
+            view.flash.assert_no_error()
+            # TODO Doesn't work due to this BZ https://bugzilla.redhat.com/show_bug.cgi?id=1489387
+            # view.flash.assert_message("Successfully deleted Saved Report from the CFME Database")
```
Comment (mshriver) [inline cfme/intelligence/reports/reports.py]: This should probably be in a test anyway, and only `assert_no_error` here

**Pull request merged**